bluesky/run_engine.py

```diff
@@ -876,6 +891,13 @@ def _close_run(self, msg):
 
     @asyncio.coroutine
     def _create(self, msg):
+        """Trigger the run engine to start collecting data for an event
+
+        Expected message object is:
+
+            Msg('create', None, name='primary')
+            Msg('create')
```

Comment:
Perhaps also note something here that is *completely* undocumented -- "name" will go into the EventDescriptor document associated with this Event.